pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -86,5 +86,7 @@ func NewResourceSyncController(
 		return nil, err
 	}
 
+	// TODO(marun) Sync the public keys used to validate bound tokens to GlobalMachineSpecifiedConfigNamespace for use by other agents
+
```

Comment:
if we don't do that, will people pick the configmap from operand namespace? Do we promote that to an API then accidently?

Reply:
Note that we have configmap sync controllers in the apiserver operators.

Reply:
What's the precedent here? When writing the enhancement I assumed that copying the config map containing public keys to the global config namespace would make it less likely that the configmap would be lost. Is that true, and still a desired goal? Or is it ok that the source of public keys be the operand configmap?

Reply:
I've updated the enhancement to have the configmap written only to the operand namespace. Please comment [there](https://github.com/openshift/enhancements/pull/150/files#diff-93464a309fcc6be1fb0cca2c92d077b2R102).

Reply:
Done